addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -16,7 +16,7 @@ def test_visitor_banner_history(self):
             'visitor_id': self.visitor.id,
             'visit_datetime': self.base_datetime,
         }, {
-            'page_id': self.env.ref('website.contactus_page').id,
+            'page_id': self.env.ref('website_form.contactus_page').id,
```

Comment:
If you do that, website_livechat would now have to depend on website_form. That's a possibility but maybe the cleaner way would be to use another **website** page (which you can also create during this test for this test's purpose... that would be even cleaner to me as it would make this test independent from future changes in website data).